pkg/kubehound/graph/edge/pod_patch.go

```diff
@@ -153,3 +156,46 @@ func (e *PodPatch) Stream(ctx context.Context, store storedb.Provider, _ cache.C
 
 	return adapter.MongoCursorHandler[podPatchGroup](ctx, cur, callback, complete)
 }
+
+/* db.permissionSets.aggregate([
```

Comment:
Is this expected to be there?

Reply:
this was the raw mongo request, will dump it somewhere else, but I think it could be useful to have all of them somewhere. Always a pain to translate golang to mongo queries.